### ISSUE TITLE
uninitialized constant Signet::OAuth2::Client::DateTime (NameError)

### ISSUE THREAD

**Issue by touhouota** [google/signet#114]
I used googleauth gem to use the Calendar API.
When I ran the command line sample code, I got a title error.
I got better at the first authentication, but an error came after the second.
I think that DateTime, which normalizes "expiration_time_millis" in token, may be the cause.
In fact, commenting out the part of DateTime no longer causes an error. (lines 1189 and 1190 of client.rb)

I think that it is necessary to require date in the signet.
``` 
8: from bug_test.rb:13:in `<main>'
7: from /Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/googleauth-0.6.6/lib/googleauth/user_authorizer.rb:136:in `get_credentials'
6: from /Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/googleauth-0.6.6/lib/googleauth/user_authorizer.rb:136:in `new'
5: from /Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/googleauth-0.6.6/lib/googleauth/user_refresh.rb:89:in `initialize'
4: from /Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/signet-0.9.0/lib/signet/oauth_2/client.rb:107:in `initialize'
3: from /Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/signet-0.9.0/lib/signet/oauth_2/client.rb:192:in `update!'
2: from /Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/signet-0.9.0/lib/signet/oauth_2/client.rb:230:in `update_token!'
1: from /Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/signet-0.9.0/lib/signet/oauth_2/client.rb:786:in `expires_at='

/Users/touhouota/google_calendar/vendor/bundle/ruby/2.5.0/gems/signet-0.9.0/lib/signet/oauth_2/client.rb:1189:in `normalize_timestamp': uninitialized constant Signet::OAuth2::Client::DateTime (NameError)
```

**Comment by TheRoyalTnetennba** [comment]
Thanks for reporting this.

**Comment by TheRoyalTnetennba** [comment]
Should be fixed on 0.9.1.  Please ping if you're still having issues.
